### ISSUE TITLE
html_notebook not show Python code chunk after run it

### ISSUE THREAD

**Issue by nietzschetmh** [rstudio/rmarkdown#1461]
Hi, i'm trying to play `reticulate` and here's the code:
```
---
title: "R Notebook"
output: html_notebook
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
    echo = TRUE,
    message = FALSE,
    warning = FALSE
    
)
```

## 0x00. Environemnt Setup
```{r Environment Setup}
library(reticulate)
conda_list()
use_condaenv('anaconda3')
```

## 0x01. ML in Python
```{python}

import numpy as np
import pandas as pd

from sklearn.model_selection import train_test_split
from sklearn.ensemble import RandomForestRegressor
from sklearn import preprocessing
from sklearn.pipeline import make_pipeline
from sklearn.model_selection import GridSearchCV
from sklearn.metrics import mean_squared_error, r2_score
from sklearn.externals import joblib



data = pd.read_csv('C:/Users/e624375/Downloads/winequality-red.csv', sep = ';')


y = data.quality
X = data.drop('quality', axis = 1)

X_train, X_test, y_train, y_test = train_test_split(
X, y, 
test_size = 0.2,
random_state = 123,
stratify = y
)

scaler = preprocessing.StandardScaler().fit(X_train)
X_test_scaled = scaler.transform(X_test)

pipeline = make_pipeline(
    preprocessing.StandardScaler(),
    RandomForestRegressor(n_estimators = 100)
)

hyperparameters = {
'randomforestregressor__max_features':['auto', 'sqrt', 'log2'],
'randomforestregressor__max_depth':[None, 5, 3, 1]
}


clf = GridSearchCV(pipeline, hyperparameters, cv = 10)
clf.fit(X_train, y_train)


print(clf.best_params_)

y_pred = clf.predict(X_test)
print(r2_score(y_test, y_pred))
print(mean_squared_error(y_test, y_pred))


```

****If i press Ctrl + Shift + K without running the python code chunk, it will quickly output nb.html file and the python code chunk is displayed.****
![good](https://user-images.githubusercontent.com/3357585/46681208-ba9be500-cbb8-11e8-82fa-9ed87e24f320.jpg)

However, if i run this python chunk by clicking the green triangle button in the chunk and rerun Ctrl + Shift + K, python code chunk is disappeared in the newly generated nb.html. 
![bad](https://user-images.githubusercontent.com/3357585/46681232-c5567a00-cbb8-11e8-904d-c004bbc92683.jpg)

Is this normal and designed to be like this?

**Comment by yihui** [comment]
Could you [minimize your example](https://yihui.name/en/2017/09/the-minimal-reprex-paradox/) and also provide your session info (per [contributing guide](https://github.com/rstudio/rmarkdown/issues/new), which you seem to have ignored)?

```r
xfun::session_info('rmarkdown')
```

Thanks!

**Comment by cderv** [comment]
Hi @nietzschetmh , 

I can't reproduce this issue with a simpler example: 
````md
---
title: "R Notebook"
output: html_notebook
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
    echo = TRUE,
    message = FALSE,
    warning = FALSE
)
```

```{r}
library(reticulate)
use_condaenv('r-reticulate')
```

```{python}
import pandas as pd
df = pd.DataFrame({
   "Name": ["Braund, Mr. Owen Harris",
            "Allen, Mr. William Henry",
            "Bonnell, Miss. Elizabeth"],
   "Age": [22, 35, 58],
   "Sex": ["male", "male", "female"]}
   )
df
```
````
The resulting `.nb.html` is including the python output.
![image](https://user-images.githubusercontent.com/6791940/97872553-856e0c80-1d16-11eb-9ac1-9cd53cc88be9.png)

This is with very last versions of all packages and RStudio IDE. 

I'll close this but If you still have issues, please reopen.
